blueprints.yaml

```diff
@@ -54,4 +54,19 @@ form:
           default:
             - image/*
           validate:
-              type: commalist
\ No newline at end of file
+              type: commalist
+    recaptcha:
+      type: section
+      title: PLUGIN_FORM.RECAPTCHA
+
+      fields:
+        recaptcha.site_key:
+          type: text
+          label: PLUGIN_FORM.RECAPTCHA_SITE_KEY
+          help: PLUGIN_FORM.RECAPTCHA_SITE_KEY_HELP
+          default: '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI'
```

Comment:
Those are the values for testing keys. But certainly that could be a little bit confusing.
Better go for empty default values.